src/components/Button/Button.tsx

```diff
@@ -105,11 +107,17 @@ class Button extends UIComponent<Extendable<ButtonProps>, ButtonState> {
      */
     onFocus: PropTypes.func,
 
+    /** A button can be formatted to show different levels of emphasis. */
+    primary: PropTypes.bool,
+
     /** A button can be formatted to show only text in order to indicate some less-pronounced actions. */
     text: PropTypes.bool,
 
+    /** The type prop specifies the type of button. */
+    type: PropTypes.oneOf(['button', 'submit', 'reset']),
```

Comment:
We don't seem to handle `type` so I think we should remove it from `propTypes` and the interface